### PR TITLE
Added additional script scenarios

### PR DESCRIPTION
**Pull request by michalpiatek**
Fixed few typos

### CODE REVIEW

File: csl-jmeter-test-plan.jmx [hunk 1]
```diff
@@ -302,6 +302,627 @@
...
+          <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="req_profile_name" enabled="true">
+            <elementProp name="HTTPsampler.Arguments" elementType="Arguments" guiclass="HTTPArgumentsPanel" testclass="Arguments" enabled="true">
+              <collectionProp name="Arguments.arguments"/>
+            </elementProp>
+            <stringProp name="HTTPSampler.domain">${BASE_URL_LPG_SERVICE}</stringProp>
+            <stringProp name="HTTPSampler.port"></stringProp>
+            <stringProp name="HTTPSampler.protocol">https</stringProp>
+            <stringProp name="HTTPSampler.contentEncoding"></stringProp>
+            <stringProp name="HTTPSampler.path">profile/given-name</stringProp>
+            <stringProp name="HTTPSampler.method">GET</stringProp>
```
Comment (charlesluokb) [inline csl-jmeter-test-plan.jmx]: would it be `PATCH` method? we've found the Hateoas calls are made in the PATCH profile/given-name

Comment (charlesluokb) [reply csl-jmeter-test-plan.jmx]: I see it now

**Comment by charlesluokb** [comment]
@michalpiatek,

Below is the table lists mappings from `profile/*` to controller methods

> 
url   | controller method  (server.js) | use Traverson.hal
/profile/otherAreasOfWork | profileController.updateOtherAreasOfWork | Y
/profile/interests | profileController.updateInterests | Y
/profile/name | profileController.updateName | Y
/profile/profession | profileController.updateProfession | Y
/profile/lineManager | profileController.updateLineManager | N
/profile/organisation | profileController.updateOrganisation | Y
/profile/grade | profileController.updateGrade | Y
/profile/line-manager | userController.tryUpdateProfile | y
/profile/department | userController.tryUpdateProfile | y
/profile/other-areas-of-work | userController.tryUpdateProfile | y
/profile/primary-area-of-work | userController.tryUpdateProfile | y
/profile/given-name | userController.tryUpdateProfile | y

In regards to the profile controller, you have added `/profile/interests`, is it worth of adding one or two more? e.g. /profile/grade for which I am guessing is called frequently

File: csl-jmeter-test-plan.jmx (hunk 1 above)
Comment (michalpiatek) [reply csl-jmeter-test-plan.jmx]: It is a patch behind the scenes - it is what a browser does:
- gets a page
- does a post with application/x-www-form-urlencoded

**Review by charlesluokb — APPROVED** [review]

**Review by cloudev7 — APPROVED** [review]

**Review by cloudev7 — APPROVED** [review]

**Review by cloudev7 — APPROVED** [review]

**Review by cloudev7 — APPROVED** [review]

**Review by cloudev7 — APPROVED** [review]

**Pull request merged**